doc/api/timers.md

```diff
@@ -94,6 +106,9 @@ set to `1`.
 If `callback` is not a function, a [`TypeError`][] will be thrown.
 
 ### setTimeout(callback, delay[, ...arg])
+<!-- YAML
+added: v0.0.1
```

Comment:
> The oldest reference I seem to be able to find of it exposed on the global (actually GLOBAL at the time) is 7a2e784 (v0.1.16)

It was present on the global before; the JS entry point (`src/node.js`) is executed without the function wrapper, so the functions defined there became part of the global scope.

If you prefer, I can go for the version in which they were added to the `timers` module, but that’s not the common usage pattern (not that it really matters too much for this old stuff anyway).